### DISCUSSION TITLE
Modelling Cattle Data

### DISCUSSION THREAD

**Discussion by bruni-cris** [Open-Systems-Pharmacology/Forum#1846] [Questions & Problems]
Hello everyone, 

I'd like to clear up some basic doubts I'm having with the cattle model in PK-sim.

1. Besides IV administration, I need to configure the subcutaneous route (SC), which is the most commonly used for administering antiparasitic drugs in this species. However, the latest version (11.3) clarifies that the cattle model should only be used for the IV route. 

Can someone explain why?
If I set up the SC route in MOBI, can I use it with any precision? 
If so, how should I configure it?

![Cattle model](https://github.com/user-attachments/assets/40e0d2e3-6852-453a-a7ec-ce5294511bbf)

2.  In Parameter Identification, the T1/2 of the formulation varied depending on the dose administered (0.05, 0.2 and 0.5 mg/Kg). In this case, shouldn't the T1/2 be the same because it is a characteristic of the formulation and not of the administration protocol? 

![T1 2](https://github.com/user-attachments/assets/dc346ecf-04c5-467e-99a5-51e546088d24)

Thank you very much for your attention!

**Comment by tobiasK2001** [comment]
Dear Bruna,

just some quick thoughts.
I cannot say to much about 1.) as I was not involved in development of the Cattle model so I can only guess here and I think there was only IV data available for validation (see here: https://github.com/Open-Systems-Pharmacology/Forum/discussions/953). Maybe the developers can comment here.

For 2.) It looks like you fitted some dissolution function  to the sc administration. Was it Weibull? / zero or first order? It is a bit hard to say without seeing the data but I think you expected that the t1/2 increases with dose as one would expect a slower release/ diffusion from a larger repository of drug after the injection for the higher doses. That is not the case in your screenshot.
So if you used a e.g. zero order infusion function for your user defined s.c. infusion you might think about using a Weibull were you have more degrees of freedom to describe your data.

B.t.w we are working on a more sophisticated SubQ model and it should be released soon (https://www.page-meeting.org/?abstract=10129)

Best, Tobias

**Reply by bruni-cris** [reply]
@tobiasK2001 thank you very much for your consideration! Initially, I made a simplified first-order formulation and hoped to optimize the formulation T1/2 after parameter identification. However, this half-life seems to be different according to the dose, as in the first image I posted. But isn't that incorrect? As it's the same formulation, I don't think the dose should influence the formulation T1/2.

Regarding what you suggested about Weibull, I tried to add it, but it's impossible to add a Weibull-type formulation when it comes to the SC route, as it doesn't appear in the dropdown menu.
![image](https://github.com/user-attachments/assets/7442fe5e-83e4-4aed-8191-c76176ead544)

Maybe I have to set up subcutaneous administration in MoBi, but I don't know how to do that yet.

**Comment by Yuri05** [comment]
"IV only" might be a bit confusing indeed. What is meant in fact is that there is **no oral absorption** model for Cattle implemented
(particularly because the 4-compartment stomach was not integrated here).
This should not prevent you from setting up a subcutaneous absorption in MoBi :)

**Reply by bruni-cris** [reply]
@Yuri05 is there any tutorial on how to set up subcutaneous absorption in MoBi?

Initially, I set up a simplified subcutaneous administration (screenshot below). Is that correct? 

![image](https://github.com/user-attachments/assets/7ac9f584-6c34-4ef9-99b4-87c65e80b948)

**Comment by PriKalra** [comment]
Hi Bruna, 

Its not surprising to see different t1/2 in some context of SubCu.. I would suggest to explore the following: 
- do you have non-linear clearance kinetics?
- With subcutaneous administration, drug distribution into systemic circulation can be influenced by local tissue binding and slower diffusion, which may prolong drug presence in peripheral compartments. As dose increases, tissue saturation may alter distribution dynamics, changing the effective half-life. As there is no oral absorption- this may be one place where there would be some answers.

**Comment by StephanSchaller** [comment]
@bruni-cris. I take it that the nonlinearity you show is what the model was fitted to and it matches the data well?

This nonlinearity can be caused by other processes (clearance, binding etc.). 
Within the model, there is no reason that your formulation or your absorption from the SC (skin interstitial) compartment should behave nonlinearly related to the dose as long as you don't implement a nonlinear process (not sth. that the default formulation functions in PK-Sim have as a characteristic).

And yes, as @Yuri05 said, the cattle model is just not made for oral, as we have not yet implemented the complex GI tract. 
We collaborate with a number of Animal Health companies and this will be added sooner or later as the need is there ;-)
